test/uk/gov/hmrc/agentfirelationship/controllers/RelationshipControllerSpec.scala

```diff
@@ -189,17 +191,19 @@ class RelationshipControllerSpec extends UnitSpec with MockitoSugar with GuiceOn
       verify(mockMongoService, times(0)).findRelationships(any(), any(), any())(any())
     }
 
-    "return Status: OK for deleting a record as a client" in {
+    "return Status: OK for deleting multiple records as a client" in {
```

Comment:
These tests have been changed to work for deleting multiple relationships at once, which is the new functionality. However, this should be added as new additional tests, not overwriting the pre-existing tests for deleting a single relationship

Reply:
Ive adapted the tests that were only client based, as the functionality that was used before is no longer in use and only what I have added will be.